gamemodes/terrortown/gamemode/server/sv_player_ext.lua

```diff
@@ -1225,10 +1234,16 @@ local plymeta_old_Give = plymeta.Give
 -- @realm server
 function plymeta:Give(weaponClassName, bNoAmmo)
 	self.forcedPickup = true
+	self.forcedGive = true
```

Comment:
I had to split them to prevent forced pickup by pressing e on a weapon but allowing giving weapons with give. Do you have any idea how to solve this in a different way?